api/app/src/main/kotlin/packit/AppConfig.kt

```diff
@@ -44,6 +44,8 @@ class AppConfig(private val environment: Environment) {
     val authEnabled: Boolean = requiredEnvValue("auth.enabled").toBoolean()
     val authGithubAPIOrg: String = requiredEnvValue("auth.githubAPIOrg")
     val authGithubAPITeam: String = requiredEnvValue("auth.githubAPITeam")
+    val authDeviceFlowExpirySeconds: Long = requiredEnvValue("auth.deviceFlowExpirySeconds").toLong()
+    val authDeviceFlowVerificationUri: String = requiredEnvValue("auth.deviceFlowVerificationUri")
```

Comment:
```suggestion
    val authDeviceFlowExpirySeconds: Long = requiredEnvValue("auth.deviceFlow.expirySeconds").toLong()
    val authDeviceFlowVerificationUri: String = requiredEnvValue("auth.deviceFlow.verificationUri")
```

I think this is clearer, but don't feel too strongly either.

If you make there a `@ConfigurationProperties` class instead, you can make the type a `Duration` and then we can use units in the property value. See for example the ServiceLoginConfig class.